public/app/plugins/datasource/loki/LanguageProvider.ts

```diff
@@ -44,13 +44,19 @@ export default class LokiLanguageProvider extends LanguageProvider {
     Object.assign(this, initialValues);
   }
 
-  request = async (url: string, params?: Record<string, string | number>, throwError?: boolean) => {
+  request = async (
+    url: string,
+    params?: Record<string, string | number>,
+    throwError?: boolean,
+    options?: Partial<BackendSrvRequest>
```

Comment:
I would consider renaming this `requestOptions`, like you do on the consumer side, to add a tiny bit of extra context ot its purpose.